### DISCUSSION TITLE
Why using different version variables in Gradle?

### DISCUSSION THREAD

**Discussion by JuJup** [quarkusio/quarkus#37326] [Q&A]
When using Quarkus with Gradle, the `quarkus create` tool creates the basic files for me. What I don't understand is, why there are two different variables for the Quarkus version in `gradle.properties`:

```
#Gradle properties
quarkusPluginId=io.quarkus
quarkusPluginVersion=3.5.3
quarkusPlatformGroupId=io.quarkus.platform
quarkusPlatformArtifactId=quarkus-bom
quarkusPlatformVersion=3.5.3

```

As far as I understood, these versions should always align anyways. So why not using a common `quarkusVersion` variable, is there a specific reason?

EDIT: The reason I ask: I'm using automated dependency updates with Renovate Bot and currently it detects (correctly) a version major update for `io.quarkus` (the plugin), but not for the platform. However, my code is still working, but it seems a bid odd though.

[a comment by an automated account is not shown]

**Comment by maxandersen** [comment]
Because those two things (core and platform) has different release cycles. 

I actually thought we had a matching gradle release as we have for maven to reduce the dependabot churn. @aloubyansky ^ ?

**Reply by JuJup** [reply]
Thanks for your reply. If they have different release cycles I guess than it's ok if they differ.
[Maven](https://mvnrepository.com/artifact/io.quarkus/quarkus-bom) and [Gradle](https://plugins.gradle.org/plugin/io.quarkus/3.6.0) Releases are the same btw., but the`io.quarkus.platform` [version](https://mvnrepository.com/artifact/io.quarkus.platform/quarkus-bom) seems to differ.

**Reply by maxandersen** [reply]
yes, the maven plugin is actually released under different gavs - io.quarkus and io.quarkus.platform/com.quarkus.redhat.com.platform - allowing simpler update mechanisms.

we didn't do it for gradle (yet). not sure if we can without breaking how gradle assumes and consumes how plugin and their plugin marketplace works.

**Reply by aloubyansky** [reply]
The Gradle plugin is released along with the Quarkus core. However, if you use RHBQ, for example, the versions of the Gradle plugin and the platform BOM will not match. The same may happen upstream too.
